tests/integration/test_datastore_storage_transfers.py

```diff
@@ -61,7 +61,7 @@ def test_enhance_and_transfer_to_drive(
         libraryId=1,
         resourceCategoryId=1,
         sourceId=1,
-        bibDate=datetime.datetime.utcnow().date(),
+        bibDate=datetime.datetime.now(timezone.utc).date(),
```

Comment:
`bibDate=datetime.datetime.now(datetime.timezone.utc).date()`

Reply:
good catch, I wonder why this was not flagged by tests or even as a syntax error?